apps/web/google_docs.py

```diff
@@ -0,0 +1,35 @@
+from talon import Context, Module, actions
+
+ctx = Context()
```

Comment:
Looks like this context doesn't match anything. This file shouldn't be overriding default implementations of actions outside of Google Docs.

Reply:
Probably just want to use a matcher like this:
```
os: mac
browser.host: docs.google.com
```
You can then probably get rid of the app declaration as well for now and just duplicate the browser.host matcher here and in the .talon. As auscompgeek mentioned you'll need to pin them both to be mac only anyway, so adding an app may not be worth it. I don't mind if you do want to define the app though.